### PR TITLE
Add new awesome recipe with tests

### PR DESCRIPTION
**Pull request by afiune**

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: recipes/awesome.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+directory '/tmp/awesome'
+
+log 'Storing awesome cookbook'
```
Comment (afiune) [inline recipes/awesome.rb:4]: Maybe just a test for this resource

**Comment by afiune** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 1914893c-c44a-48f5-8532-6b2862abcca6 approved by: @afiune

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Comment by afiune** [comment]
@delivery deliver

**Comment by chef-delivery** [comment]
Change: 1914893c-c44a-48f5-8532-6b2862abcca6 delivered by: @afiune

**Pull request merged**